grpc/interceptors.go

```diff
@@ -13,21 +16,59 @@ import (
 	"github.com/letsencrypt/boulder/features"
 )
 
+const (
+	returnOverhead         = 20 * time.Millisecond
+	meaningfulWorkOverhead = 100 * time.Millisecond
+	clientRequestTimeKey   = "client-request-time"
+	serverLatencyKey       = "server-latency"
+)
+
 // serverInterceptor is a gRPC interceptor that adds Prometheus
 // metrics to requests handled by a gRPC server, and wraps Boulder-specific
 // errors for transmission in a grpc/metadata trailer (see bcodes.go).
 type serverInterceptor struct {
 	serverMetrics *grpc_prometheus.ServerMetrics
+	rpcLag        prometheus.Histogram
```

Comment:
Rather than separately defining `serverMetrics` and `rpcLag` fields on serverInterceptor, why not embed a field of type `serverMetrics` into `serverInterceptor`?